Connectivity/Classes/Connectivity.swift

```diff
@@ -177,21 +180,23 @@ public extension Connectivity {
         let totalChecks: Int = connectivityURLs.count
         
         // Connectivity check callback
-        let completionHandler: (Data?, URLResponse?, Error?) -> Void = {  [weak self] (data, response, error) in
-            let connectivityCheckSuccess = self?.connectivityCheckSucceeded(data: data) ?? false
-            connectivityCheckSuccess ? (successfulChecks += 1) : (failedChecks += 1)
-            dispatchGroup.leave()
-            // Abort early if enough tasks have completed successfully
-            self?.cancelConnectivityCheck(pendingTasks: tasks, successfulChecks: successfulChecks,
-                                          totalChecks: totalChecks)
+        let completionHandlerForUrl: (URL) -> ((Data?, URLResponse?, Error?) -> Void) = { url in
```

Comment:
To include the URL in response validation, the signature here was updated to require passing the URL, so that it could be captured and included when moving on to response validation